app/routes.py

```diff
@@ -1,2 +1,196 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, jsonify
+from datetime import datetime
+import requests
+from app import slack_key
+from app.models.goal import Goal
 
+task_list_bp = Blueprint("task_list", __name__, url_prefix = "/tasks")
+
+@task_list_bp.route("", methods = ["POST"])
+def create_task():
+    request_body = request.get_json()
+    attributes = ["title", "description", "completed_at"]
+
+    for attribute in attributes:
+        if attribute not in request_body:
+
+            return make_response({"details": "Invalid data"}), 400
+
+    new_task = Task(title = request_body["title"],
+                description = request_body["description"],
+                completed_at = request_body["completed_at"])
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    return jsonify({"task": new_task.to_json()}), 201
+
+
+@task_list_bp.route("", methods = ["GET"]) 
+def get_all_tasks():
+    # tasks = Task.query.all()
+    sort_method = request.args.get("sort")
+    if sort_method == "asc":
+        tasks = Task.query.order_by(Task.title.asc())
+        # SELECT * FROM task ORDER BY title ASC
+    elif sort_method == "desc":
+        tasks = Task.query.order_by(Task.title.desc())
+    else:
+        tasks = Task.query.all()
+    task_response = []
+    for task in tasks:
+        task_response.append(task.to_json())
+
+    return jsonify(task_response), 200
+
+
+@task_list_bp.route("/<task_id>", methods = ["GET", "DELETE", "PUT"])
```

Comment:
For the route "" the "GET" and "POST" methods are split into different functions, while multiple methods are combined for this route.  Either way is fine, but I recommend using a consistent style throughout.